.github/workflows/build-apps.yml

```diff
@@ -130,13 +130,14 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - os: macos-14
+          - os: namespace-profile-macos-6-cores
             platform: mac
```

Comment:
I assume naming these `macos`, `windows`, and `ubuntu` would break something else? I'm merely nitpicking to get consistency with the way test jobs are named. 😄

Reply:
Believe me I want that as well hahaha. But I'd rather do it seperate because it impacts this build-apps workflow as well as publish-apps-release

Reply:
I'll do it tho!

Reply:
@pierremtb Nice! And feel free to change `ubuntu` to `linux` across the board if that makes more sense for builds.

Reply:
https://github.com/KittyCAD/modeling-app/pull/7427

Reply:
So these ubuntu-generated artifacts are meant to be compatible across all linux distros with the AppImage bundle right? So I guess `out-{arch}-linux` still makes sense. But feel free to review and chime in there